shared/teams/role-picker/index.tsx

```diff
@@ -103,15 +105,16 @@ type RoleRowWrapperProps = {
   onSelect?: () => void
   disabledReason?: string
   role: Role<true>
+  plural: boolean
 }
 const RoleRowWrapper = (props: RoleRowWrapperProps) => {
-  const {role, selected, onSelect, disabledReason} = props
+  const {role, selected, onSelect, disabledReason, plural} = props
   const roleInfo = rolesMetaInfo(role)
 
   // @ts-ignore spring is confused that I'm animating different things on desktop vs mobile
   const style = useSpring({
     ...(Styles.isMobile ? {flexGrow: selected ? 1 : 0} : {height: selected ? 160 : 42}),
-    config: {tension: Styles.isMobile ? 215 : 230},
+    config: {tension: Styles.isMobile ? 250 : 260},
```

Comment:
@cecileboucheron this controls the animation speed, higher is faster. I made it a little faster but feel free to play with it and choose a different value.